config/locales/en.yml

```diff
@@ -239,7 +242,11 @@ en:
         custom_select_error: Select yes if your basic care needs are being met at the moment
       dietary_requirements:
         title: Do you have any special dietary requirements?
-        hint: For example, you're a vegetarian or you have a food allergy.
+        hint: |
+          <p class="govuk-hint app-hint">For example, you're a vegetarian or you have a food allergy.</p>
+          <div class="govuk-inset-text">
```

Comment:
The component seems  wrap all this code with a div and govuk-body class which I did not expect.

So I think your previous revision was fine, sorry :man_facepalming:

Reply:
No worries - I've reverted it